### DISCUSSION TITLE
Alerts Actions suggestion - AbuseIPDB Search

### DISCUSSION THREAD

**Discussion by greatapoc** [Security-Onion-Solutions/securityonion#2541] [Ideas] [answered]
Hey guys just a quick suggestion for an additional item under the Actions menu in Alerts/Hunt. I often check addresses against AbuseIPDB to see if it is a known malicious IP. I'm currently just copying and pasting so it's certainly no great imposition but I just thought it would be good to do it directly from SO.

**Comment by weslambert** [comment, marked as answer]
Hi @greatapoc ,

Thanks for the suggestion!  In the meantime, you can actually copy `/opt/so/saltstack/default/salt/soc/files/soc/alerts.actions.json` to `/opt/so/saltstack/local/salt/soc/files/soc/alerts.actions.json` and modify the entries there to include a lookup for AbuseIPDB.

Like so:

`{ "name": "actionAbuseIPDB", "description": "actionAbuseIPDBHelp", "icon": "fa-external-link-alt", "target": "_blank","links": [ "https://www.abuseipdb.com/check/{value}"`

Then restart SOC with `so-soc-restart`

**Reply by greatapoc** [reply]
Hi @weslambert 

That's great! Thanks so much for that I'll have a crack at it now.

**Reply by greatapoc** [reply]
That was super easy and it works perfectly. I just changed the name from actionAbuseIPDB to AbuseIPDB. Thanks again.

**Reply by weslambert** [reply]
Awesome, yeah sorry, hadn't had a chance to test before sending, but glad to hear it's working! 👍

**Reply by greatapoc** [reply]
No worries at all you're taking the time out from other more important activities I'm sure. Much appreciated.
